trackio/run.py

```diff
@@ -1,18 +1,25 @@
-from trackio.storage import TrackioStorage
+from gradio_client import Client
+
 from trackio.utils import generate_readable_name
 
 
 class Run:
     def __init__(
-        self, project: str, name: str | None = None, config: dict | None = None
+        self,
+        project: str,
+        client: Client,
+        name: str | None = None,
+        config: dict | None = None,
     ):
         self.project = project
+        self.client = client
         self.name = name or generate_readable_name()
         self.config = config or {}
-        self.storage = TrackioStorage(project, name, config)
 
     def log(self, metrics: dict):
-        self.storage.log(metrics)
+        self.client.predict(
```

Comment:
Is it called `predict` because this API was originally used to run model predictions? I guess it's more of a generic server-API-calling method now?

Reply:
Exactly